Automattic/vip-go-mu-plugins
# Update Jetpack 9.8 to 9.8.1

## Description

This PR bumps the Jetpack 9.8 branch to 9.8.1. The update has been done by running:

```
bin/update-jetpack-subtree.sh 9.8 9.8.1
```

## Changelog Description

### Plugin Updated: Jetpack 9.8.1

We upgraded Jetpack 9.8 to Jetpack 9.8.1.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Run it on a site with Jetpack enabled.
3. Check that Jetpack runs fine and it shows version 9.8.1
